fastautoml/fastautoml.py

```diff
@@ -1688,39 +1697,41 @@ def score(self, X, y):
         
         # TODO: Check that we have a model trained
         
-        if self.viu is None:
+        check_is_fitted(self, '_X')
+        
+        if self._viu is None:
             msdX = X
         else:
-            msdX = X[:,np.where(self.viu)[0]]        
+            msdX = X[:,np.where(self._viu)[0]]        
         
-        return self.model.score(msdX, y)
+        return self._model.score(msdX, y)
 
 
-    def MultinomialNB(self):
+    def _MultinomialNB(self):
```

Comment:
I am not sure about these changes. The idea is that you can call AutoClassifer.fit(auto=False) and then call individualy one of the supported models. So I expect these functions to be public.